test/pylib/manager_client.py

```diff
@@ -186,6 +186,10 @@ async def server_update_config(self, server_id: ServerNum, key: str, value: obje
         await self.client.put_json(f"/cluster/server/{server_id}/update_config",
                                    {"key": key, "value": value})
 
+    async def server_change_ip(self, server_id: ServerNum) -> None:
+        """Change server IP address. Applicable only to a stopped server"""
+        await self.client.put_json(f"/cluster/server/{server_id}/change_ip", {})
```

Comment:
I'd add `self._driver_update()`

Reply:
The IP address change has to happen while the node is down. So there is no need to refresh the driver at that point - and that would be useless. When the down node is started with a new ip, the driver will get a gossiper notification and will naturally get updated. Or will react to start() event on that node.